src/main/resources/config.example

```diff
@@ -0,0 +1,5 @@
+*Please create a config.properties file (default path: src/main/java/resources/config.properties) and define paths to both webdrivers as shown below:
+CHROME_DRIVER = please/put/your/path/to/chromedriver.exe
+FIREFOX_DRIVER = please/put/your/path/to/geckodriver.exe
+BASE_URL=https://automationteststore.com/
+GLOBAL_WAIT=10
```

Comment:
You are right, but for now it is just example setting, because there is no test implemented yet. But there is a small chance that in a future implementation we may forget about this setting so I am changing it to 3 seconds :)